kompendium-playground/src/main/kotlin/org/leafygreens/kompendium/playground/Main.kt

```diff
@@ -73,12 +78,29 @@ fun main() {
   ).start(wait = true)
 }
 
+var featuresInstalled = false
 fun Application.mainModule() {
-  install(ContentNegotiation) {
-    jackson {
-      enable(SerializationFeature.INDENT_OUTPUT)
-      setSerializationInclusion(JsonInclude.Include.NON_NULL)
+  // only install once in case of auto reload
```

Comment:
Ahh I was wondering why you were doing this in the earlier commits, that's good to know, I haven't used Ktors auto reload yet, this probably would have been a headache for me down the line 😄 Kinda surprised they haven't build that into the functionality itself

Reply:
yes, it is probably how you run it? I was running it using the main function and then the ktor debug property as defined in gradle was set